### PR TITLE
Refactor nuget section, add tip and trics and templates.

### PR DESCRIPTION
**Pull request by didiyordanova**

### CODE REVIEW

**Comment by didiyordanova** [comment]
do not merge! changes needs to be done.

File: installation/nuget/nuget-keys.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+title: Setting Up in .NET CLI
```
Comment (yordan-mitev) [inline installation/nuget/nuget-keys.md:2]: I suggest keeping the title aimed at CI as it was in the original article as the focus of the article is CI and NuGet keys and not so much the .NET CLI, which is the instrument to download NuGet packages.

File: installation/nuget/nuget-keys.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        </MyTelerikFeed>
+    </packageSourceCredentials>
+</configuration>
+```
+
+**2.** Set the `MY_API_KEY` environment variable by using the value of your pipeline/workflow secret.
+
+The exact steps to set the `MY_API_KEY` environment variable depend on your workflow. For more details, refer to the [Announcing NuGet Keys](https://www.telerik.com/blogs/announcing-nuget-keys) blog post by Lance McCarthy.
+
+### Using Only CLI Commands
```
Comment (yordan-mitev) [inline installation/nuget/nuget-keys.md:88]:
```suggestion
### Using .NET CLI Commands
```

File: installation/nuget/visual-studio.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  ![Package Sources field with the checked Telerik NuGet option](../../installation/images/nuget-vs-telerik-server.png)
+
+You have successfully added the Telerik NuGet feed as a Package source.
+
+
+## Step 2: Install the Telerik .NET MAUI NuGet Package
+
+The next steps describe how to authenticate your local NuGet instance and display the available packages:
+
+1. Create a new project or open an existing project.
```
[the remark this answers is not in the document]
Comment (didiyordanova) [reply installation/nuget/visual-studio.md:45]: my mistake. 
`1. .NET MAUI installed on the machine.` should be set at line 17. 
So here we keep the step `1. Create a new .NET MAUI project or open an existing project`

File: installation/nuget/visual-studio.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+1. Select **Manage NuGet Packages for Solution...**
+1. Select the Telerik NuGet **Package source** from the drop-down list.
+1. Click on the **Browse** tab to see the available packages.
+1. Enter your Telerik credentials in the Windows Authentication dialog.
+1. In the Visual Studio Package Manager, you will see all packages that are licensed to your user account.
+1. Search for the `Telerik.UI.for.Maui` or `Telerik.UI.for.Maui.Trial` package and select it.
+1. Choose the projects which require the package.
+1. Select the desired version and click **Install**.
+
+![Manage Packages for Solutions dialog with the search field and the Telerik.UI.for.MAUI package](../../installation/images/maui-nuget.png)
```
[the remark this answers is not in the document]
Comment (didiyordanova) [reply installation/nuget/visual-studio.md:56]: agree, will change the image.

File: installation/nuget/visual-studio.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  ![Telerik NuGet Package Manager context menu with the Package Manager Settings option](../../installation/images/nuget-vs-pm-settings.png)
+
+1. Select **Package Sources**, and then click the + button.
+
+  ![Package Sources dialog with the Available package sources field](../../installation/images/nuget-vs-add-source.png)
+
+1. In the **Source** field, enter `https://nuget.telerik.com/v3/index.json`. If you use a locally available NuGet package downloaded from <a href="https://www.telerik.com/account/" target="_blank">your account</a>, add the path to the local package instead of the URL.
+1. Click **Update** and then **OK**.
+
+  ![Package Sources field with the checked Telerik NuGet option](../../installation/images/nuget-vs-telerik-server.png)
```
[the remark this answers is not in the document]
Comment (didiyordanova) [reply installation/nuget/visual-studio.md:36]: image changed

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**